extensions/wikia/CreateNewWiki/templates/CreateNewWiki_Index.php

```diff
@@ -203,7 +185,7 @@
 	</ol>
 	<ul id="StepsIndicator">
 		<?php
-			$steps = $isUserLoggedIn ? 4 : 5;
+			$steps = 4;
```

Comment:
yeah, could we improve the code here and explicitly state what kind of steps are these?